src/CartLatticeExecutor.h.Rt

```diff
@@ -0,0 +1,55 @@
+<?R
+        source("conf.R")
+c_header();
+?>
+/** \file CartLatticeExecutor.h
+*/
+
+#ifndef CARTLATTICEEXECUTOR_H
+#define CARTLATTICEEXECUTOR_H
+
+#include "CartLatticeContainer.h"
+#include "LatticeData.hpp"
+
+class CartLatticeExecutor {
+  CartLatticeContainer m_container;
+  LatticeData          m_data;
+
+public:
+  template < eOperationType I, eCalculateGlobals G, eStage S >
+  void RunBorder(CudaStream_t stream) const;
+  template < eOperationType I, eCalculateGlobals G, eStage S >
+  void RunInterior(CudaStream_t stream) const;
+  void Color(uchar4 *out_ptr);
+
+  CartLatticeContainer&       getContainer()       { return m_container; }
```

Comment:
I woulde remove the getters, and the `m_` (as it's not used in the code elsewhere).

Reply:
Closed in cd5e88a59be1e4ba72f543c06f6e35e1c8c91aad